rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -22,13 +22,30 @@ module WebDriver
     module Remote
       class Bridge
         autoload :COMMANDS, 'selenium/webdriver/remote/bridge/commands'
+        autoload :LocatorConverter, 'selenium/webdriver/remote/bridge/locator_converter'
+
         include Atoms
 
         PORT = 4444
 
         attr_accessor :http, :file_detector
         attr_reader :capabilities
 
+        class << self
+          attr_accessor :extra_commands
+          attr_writer :locator_converter
+
+          def add_command(name, verb, url, &block)
```

Comment:
Would we want Selenium to use this as part of `Features` modules, so we aren't subclassing `#commands`?

Reply:
Which is to say, we probably should be doing the same thing with custom browser features as Appium does, either way.

Reply:
Yeah, I would refactor Selenium after Appium approves the changes.